### DISCUSSION TITLE
I would like to know the history of MDN

### DISCUSSION THREAD

**Discussion by debiru** [mdn/mdn-community#470] [Content] [answered]
Is there any literature that describes in detail the history as MDN Web Docs?

I know of the following articles:

- https://developer.mozilla.org/en-US/docs/MDN/At_ten/History_of_MDN
- https://en.wikipedia.org/wiki/MDN_Web_Docs
- https://hacks.mozilla.org/2021/02/mdn-localization-update-february-2021/

I hope an updated version of articles like The [History of MDN](https://developer.mozilla.org/en-US/docs/MDN/At_ten/History_of_MDN) will be published. And I expect the following to be explained:

- When was it called the "Mozilla Developer Center", "Mozilla Developer Network" and "MDN Web Docs"?
- History of MDN's system." What is "Netscape DevEdge", "MediaWiki", "DekiWiki", "Kuma" and "Yari"? When were they used?
- The time period about 2020 when the localized version was impossible to edit as a frozen state when MDN was moved to GitHub. When that freeze was lifted, and what that freeze was for.

So here is my questions:

- (1) When was [History of MDN](https://developer.mozilla.org/en-US/docs/MDN/At_ten/History_of_MDN) written?
- (2) Are there any plans to have an updated version of the History of MDN (reflecting the situation after 2023) written as a page in the MDN Web Docs?
- (3) Regarding what I want to know above, can you give me information on that?

I would be grateful if you could answer my questions.

**Comment by Rumyra** [comment, marked as answer]
Hi @debiru thanks for the question - the current 'History of MDN' page was written when MDN was ten which was in 2015. Currently we have no task to update this page, however due you highlighting interest we'll consider a more up to date version of the history including the past eight years. Hope that helps!

**Reply by debiru** [reply]
Thank you for your answer. I think several of you are interested in historical information about MDN.

I am very glad that you will consider updating that information.

**Reply by bsmth** [reply]
__update:__ there's an About page here: https://developer.mozilla.org/en-US/about :)
